src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -202,17 +192,6 @@ public function iTryToOpenCheckoutCompletePage()
     }
 
     /**
-     * @Given /^(this user) bought(?: this product| those products)$/
-     */
-    public function thisUserBought(ShopUserInterface $user)
-    {
-        $this->sharedSecurityService->performActionAsShopUser($user, function () {
```

Comment:
I think we can also remove `performActionAsShopUser` method from `sharedSecurityService`, it's not used anywhere now.

PS. Sorry for being annoying 😄